low-level/sensors/impl/centos_7/disk_monitor.py

```diff
@@ -98,6 +98,12 @@ def name():
         """@return: name of the module."""
         return DiskMonitor.SENSOR_NAME
 
+    @staticmethod
+    def impact():
+        """Returns impact of the module."""
+        return ("Disk insertion, removal and health can not be monitored "

```

Comment:
disk monitor doesnt monitor storage enclosure disks

Reply:
In recent commit, I simplified the statement.

Reply:
Done.